src/CMakeLists.txt

```diff
@@ -216,9 +216,9 @@ endif()
 set(CMAKE_C_FLAGS "-std=c99 -pipe ${WARN_FLAGS} -D_FILE_OFFSET_BITS=64 ${HARDENING_FLAGS} ${DEBUG_FLAGS} ${CMAKE_C_FLAGS} -DHAVE_POLL_H ${SQLITE_DEFINES}")
 
 set(CMAKE_C_FLAGS_DEBUG "-O0 -g3")
-set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG")
+set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG -funroll-loops -flto")
```

Comment:
Is unrolling the loops here a good idea?

On earlier pis with their comparatively smaller caches it might slow things down. I'd also wonder about the increase in memory usage on systems like that.